cmd/pke/app/phases/runtime/kubernetes/kubelet_kernel_params.yaml.tmpl

```diff
@@ -1,3 +1,6 @@
 vm.overcommit_memory=1
+# vm.overcommit_ratio=100
+vm.oom_kill_allocating_task=1
 kernel.panic=10
 kernel.panic_on_oops=1
+vm.swappiness=0
```

Comment:
I don't see a reason to set this. Normally we don't even have swap volumes, but if we do, it may be a smart decision to swap out pages that well be never accessed to gain place for fs cache. I would not change these settings without well documented or own measurements.